M365/src/DLT365Groupsupgrade.ps1

```diff
@@ -121,9 +121,9 @@ Function Debuggroupnesting {
     $ParentDGroups = @()
     try {
         $alldgs = Get-DistributionGroup -ResultSize unlimited -ErrorAction Stop
-        $CurrentDescription = "Retrieving All DGs in the EXO directory"
-        $CurrentStatus = "Success"
-        log -Function "Retrieve All DGs" -CurrentDescription $CurrentDescription -CurrentStatus $CurrentStatus
+        #$CurrentDescription = "Retrieving All DGs in the EXO directory"
```

Comment:
Did you intend to leave these lines commented out? Can we remove them if they are not needed?

Reply:
No I didn't intend to comment these lines, I uncommented them and pushed the change